rewrite-maven/src/main/java/org/openrewrite/maven/internal/MavenPomDownloader.java

```diff
@@ -792,8 +792,7 @@ Collection<MavenRepository> distinctNormalizedRepositories(
                             }
                         }
                     }
-                    if (normalized == null && !(t instanceof HttpSenderResponseException &&
-                                                ((HttpSenderResponseException) t).getBody().contains("Directory listing forbidden"))) {
+                    if (normalized == null) {
```

Comment:
We implement `repositoryAccessFailed` in a couple places, including the CLI, to warn about a failure to access a repository. Not being allowed to list directories seems like a valid setting that should not trigger that logged statement; can we determine those cases without a body?

Reply:
As far as I can tell normalized is never going to be `null` in the case that the body would contain `Directory listing forbidden` which returns with a 403 and means `e.isServerReached()` returns `true`

```
if (t instanceof HttpSenderResponseException) {
    HttpSenderResponseException e = (HttpSenderResponseException) t;
    // response was returned from the server, but it was not a 200 OK. The server therefore exists.
    if (e.isServerReached()) {
        normalized = repository.withUri(httpsUri);
    }
}
```